tests/phpunit/includes/links-trait.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Trait to ease tests made with WP installed in a subfolder.
+ */
+trait PLL_Links_Trait {
+	/**
+	 * Permalink structure.
+	 *
+	 * @var string
+	 */
+	protected $structure = '/%postname%/';
+
+	/**
+	 * Original `siteurl` option value.
+	 *
+	 * @var string
+	 */
+	protected $orig_siteurl = '';
+
+	/**
+	 * Filters `plugins_url` because `WP_CONTENT_URL` is already defined before the option `siteurl` is changed during tests for directory.
+	 * Also remove full path `POLYLANG_DIR` added to flag URL.
+	 *
+	 * @return void
+	 */
+	protected function filter_plugins_url() {
+		$this->orig_siteurl = get_option( 'siteurl' );
```

Comment:
Do you need the property? Or could you do something like:
```php
function ( $url ) use ( $original_siteurl ) {
```

Reply:
Good idea, done in aefaa523.